data/influxdb_urls.yml

```diff
@@ -5,6 +5,8 @@ oss:
       regions:
         - name: localhost:9999
           url: http://localhost:9999
+    - name: Custom
```

Comment:
should line 8 be indented more?

Reply:
No, this is a special case that populates the placeholder for the custom URL field.